app/views/layouts/application.html.erb

```diff
@@ -14,9 +14,9 @@
     <% flash.each do |name, msg| %>
       <%= content_tag :div, msg, :id => "flash_#{name}" %>
     <% end %>
-		
-		<%= yield %>
 
+		<%= yield %>
+    <%= debug(params) if Rails.env.development? %>
```

Comment:
It gives us a box on the bottom of each page in development mode. That box gives us the action that we had hit to get to that current page, and the controller that we came from, and params. It only shows up in development mode. We use the web-console gem. It basically makes our lives easier when debugging. Check out Hartl chapter 7.